aztk/config.py

```diff
@@ -11,11 +11,16 @@ def __init__(self):
         self.batch_account_name = None
         self.batch_account_key = None
         self.batch_service_url = None
+        self.batch_resource_url = None
```

Comment:
Probably not worth having account name && resource url. Maybe not worth making the change yet, but we should only require resource url since it is more verbose. We can parse out the string if using SharedKey

Reply:
I agree that this does feel like duplication, but from the user's point of view, they only need to enter one or the other.
A benefit of having account name for shared key auth is that all the information necessary for configuring batch auth is together on one screen in the web UI, so requiring the resource URL would make set up more difficult for shared key users.